js/pluralizer.js

```diff
@@ -0,0 +1,8 @@
+let thing = 'Cat';
+let count = 1;
+
+if(count < 2) {
+console.log(count + " " + thing )
+} else if (count > 1) {
+    console.log(count + " " + thing + "'s")
+}
```

Comment:
what if i have 0 cats? ( which is unacceptable and will never happen I love my cats)